### PR TITLE
allow php7

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

**Comment by prolic** [comment]
depends on https://github.com/alcaeus/mongo-php-adapter/pull/118

File: src/MongoDbEventStoreAdapter.php [hunk 1]
```diff
@@ -173,9 +173,15 @@ public function appendTo(StreamName $streamName, Iterator $streamEvents)
 
         try {
             $insertBatch->execute();
-        } catch (\MongoWriteConcernException $e) {
+        } catch (\MongoDuplicateKeyException $e) {
             throw new ConcurrencyException('', 0, $e);
```
Comment (codeliner) [inline src/MongoDbEventStoreAdapter.php]: We should add a short exception message so that one knows what went wrong when the exception is logged.

Comment (prolic) [inline src/MongoDbEventStoreAdapter.php]: would be same for doctrine-adapter, suggestions?

**Comment by prolic** [comment]
see also https://github.com/alcaeus/mongo-php-adapter/pull/120/

**Comment by prolic** [comment]
ping @codeliner

**Pull request merged**